agent/stats/common_test.go

```diff
@@ -247,9 +250,19 @@ func validateEmptyTaskHealthMetrics(t *testing.T, engine *DockerStatsEngine) {
 }
 
 func createFakeContainerStats() []*ContainerStats {
+	netStats := &NetworkStats{
+		rxBytes:   796,
+		rxDropped: 6,
+		rxErrors:  0,
+		rxPackets: 10,
+		txBytes:   8192,
+		txDropped: 5,
+		txErrors:  0,
+		txPackets: 60,
+	}
 	return []*ContainerStats{
-		{22400432, 1839104, uint64(0), uint64(0), parseNanoTime("2015-02-12T21:22:05.131117533Z")},
-		{116499979, 3649536, uint64(0), uint64(0), parseNanoTime("2015-02-12T21:22:05.232291187Z")},
+		{22400432, 1839104, uint64(0), uint64(0), netStats, parseNanoTime("2015-02-12T21:22:05.131117533Z")},
+		{116499979, 3649536, uint64(0), uint64(0), netStats, parseNanoTime("2015-02-12T21:22:05.232291187Z")},
```

Comment:
good change.  I'll actually need to change the two `uint64(0)` to be `storageReadStats` and `storageWriteStats`.